CalibCalorimetry/HcalTPGAlgos/src/HcaluLUTTPGCoder.cc

```diff
@@ -99,7 +102,8 @@ void HcaluLUTTPGCoder::init(const HcalTopology* top, const HcalTimeSlew* delay)
   nHFEta_ = (lastHFEta_ - firstHFEta_ + 1);
   maxDepthHF_ = topo_->maxDepth(HcalForward);
   sizeHF_ = 2 * nHFEta_ * nFi_ * maxDepthHF_;
-  size_t nluts = (size_t)(sizeHB_ + sizeHE_ + sizeHF_ + 1);
+  sizeZDC_ = 2 * 9;
```

Comment:
Yes, that would be preferred. However, this number is not defined in the geometry yet, since that is still being worked on by the HCAL geometry expert.